common/countries/CrystalEmpire.txt

```diff
@@ -18,38 +18,38 @@ historical_idea_groups = {
 }
 
 monarch_names = {
-	"Apple #0" = 20
-	"Rainbow #0" = 20
-	"Twilight #0" = 20
-	"Pinkie #0" = 20
-	"Rosie #0" = 15
-	"Sweety #0" = 15
+	"Apple #0" = -20
```

Comment:
Because female rulers. It was just odd the first time I got a King Twilight, as a ruler, for instance.